run.sh

```diff
@@ -32,6 +32,25 @@ cp /root/temp/modules/*.omod $OPENMRS_MODULES
 rm -rf ${OPENMRS_HOME}/${OPENMRS_NAME}/.openmrs-lib-cache/
 echo "Modules copied."
 
+#Configure Bahmni appointments UI
+wget -q https://bintray.com/openmrs/owa/download_file?file_path=appointments-1.0.0.zip -O /usr/local/tomcat/appointments.zip
+unzip /usr/local/tomcat/appointments.zip -d /usr/local/tomcat/webapps/appointments
+rm -f /usr/local/tomcat/appointments.zip
+cp /root/temp/bahmniapps/config/appointments/ng-constants.json /usr/local/tomcat/webapps/appointments/constants/ng-constants.json
+cp /root/temp/bahmniapps/i18n /usr/local/tomcat/webapps/appointments/
+cp /root/temp/bahmniapps/config /usr/local/tomcat/webapps/appointments/bahmniapps
+#Configure Bahmni Role
+mysql -u${OPENMRS_DB_USER} -p${OPENMRS_DB_PASS} ${DB_NAME} -e "INSERT INTO global_property(property, property_value, description, uuid) values ('bahmni.appointments.runningOnOpenMRS', 'true', 'If set to true, the appointments ui will run independent of bahmni core', uuid());INSERT INTO global_property(property, property_value, description, uuid) values ('bahmni.primaryIdentifierType', '05a29f94-c0ed-11e2-94be-8c13b969e334', 'Primary identifier type for looking up patients, generating barcodes, etc.', uuid());"
```

Comment:
I would first check if these db values exists, before inserting. I would also combine them into a sql script file then run it as a file.

Reply:
isn't L12 checking for this already?

Reply:
I guarantee you that it will create multiple duplicate values every time a new container is created. That's because the primary key is not part of your inserts.

Reply:
Alright, let me make the improvements

Reply:
@nkimaina Actually we will not get duplicated since we have a uuid and composite keys in the tables. So do you think using `INSERT IGNORE` would be appropriate?